composer.json

```diff
@@ -28,7 +28,7 @@
         "imagine/imagine": "~0.6.1",
         "massive/search-bundle": "0.16.*",
         "sensio/framework-extra-bundle": "^3.0",
-        "sulu/document-manager": "~0.9.1",
+        "sulu/document-manager": "dev-develop",
```

Comment:
its done in this PR - see https://github.com/sulu/sulu/pull/3306/files#diff-76d3c067d4736e419fcb44d7d23cbd54 and also https://github.com/sulu/sulu-document-manager/pull/110